From 6a792f0c655adf383671e01e6554c4e243ac24ad Mon Sep 17 00:00:00 2001
From: ramana <ramana@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Fri, 6 May 2011 10:21:26 +0000
Subject: [PATCH] 2011-05-06  Ramana Radhakrishnan  <ramana.radhakrishnan@linaro.org>

        PR target/48252
        * config/arm/neon.md (neon_vtrn<mode>): Fix typo
        from earlier commit.



git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_6-branch@173480 138bc75d-0d04-0410-961f-82ee72b054a4

index 491dd98..fddd632 100644
--- a/gcc/config/arm/neon.md
+++ b/gcc/config/arm/neon.md
@@ -4093,7 +4093,7 @@
                     (const_string "neon_bp_3cycle")))]
 )
 
- "neon_vtrn<mode>"
+(define_expand "neon_vtrn<mode>"
   [(match_operand:SI 0 "s_register_operand" "r")
    (match_operand:VDQW 1 "s_register_operand" "w")
    (match_operand:VDQW 2 "s_register_operand" "w")]
-- 
1.7.0.4

